packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -192,11 +194,16 @@ class HorizontalOverflow extends Foundation<
             return;
         }
 
-        const itemsHeight: number = this.getItemMaxHeight();
+        const itemsHeight: number =
+            this.props.fixedHeight !== null
+                ? this.props.fixedHeight
+                : this.getItemMaxHeight();
```

Comment:
You can just add this logic into `getItemMaxHeight` instead of implementing this same logic twice here and in` itemsOnLoad`

Reply:
My concern with that was the function name: getItemMaxHeight(). This doesn't imply it was return the fixed height and someone may want to use it in the future to actually get the max height of the child items. I could rename it to something like getFixedHeightOrItemMaxHeight, or add another function called getHorizontalOverflowHeight that performs this ternary check

Reply:
I think I'd prefer then that we go with your suggestion of an additional method just named `getItemHeight` where this ternary is performed and use that here.

Reply:
Added 👍